patches/net/minecraft/world/item/Item.java.patch

```diff
@@ -186,3 +204,19 @@
              if (this.components == null) {
                  this.components = DataComponentMap.builder().addAll(DataComponents.COMMON_ITEM_COMPONENTS);
              }
+@@ -391,10 +_,13 @@
+ 
+         DataComponentMap buildAndValidateComponents() {
+             DataComponentMap datacomponentmap = this.buildComponents();
++            validateComponents(datacomponentmap);
++            return datacomponentmap;
++        }
++
++        public static void validateComponents(DataComponentMap datacomponentmap) {
```

Comment:
Making it return `DataComponentMap` (even though we don't use it) would remove part of the hunk.